test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/acquisitiondetails/AcquisitionDetailsControllerSpec.scala

```diff
@@ -2514,5 +2520,52 @@ class AcquisitionDetailsControllerSpec
       BAD_REQUEST
     )
   }
+}
+
+object AcquisitionDetailsControllerSpec extends AcquisitionDetailsControllerSpec {
```

Comment:
do you need `extends AcquisitionDetailsControllerSpec` here? What is it doing for you here?